src/GovUk.Education.ExploreEducationStatistics.Data.Model/Country.cs

```diff
@@ -1,14 +1,15 @@
+#nullable enable
 namespace GovUk.Education.ExploreEducationStatistics.Data.Model
 {
-    public class Country : ObservationalUnit
+    public class Country : ObservationalUnit, ILocationAttribute
     {
-        public Country(string code, string name) : base(code, name)
+        public Country(string? code, string? name) : base(code, name)
```

Comment:
What's the driver behind making these nullable now?

Reply:
There should be no overall change here and I'm not trying to rework anything except to represent their current nullable state correctly in the model after setting `Location` and the attribute types (e.g. Country) as nullable reference types with `nullable enable`. 

In the example of Country, Location `Country_Code` and `Country_Name` fields are nullable in the database so they have now been marked as nullable fields in the `Location` type.

You can always 'get' Location.Country but if the values are not provided then it's constructed with a null code and name so these constructor params are nullable too.

If I don't make this change to the `Location` type I think I would get a generated migration altering all the fields of the table to be not null. I have checked to make sure that no migration code is generated with these changes.

If I make the change to the `Location` type but didn't make this change to the attribute types (e.g. the change to `Country` here) then we get code analysis warnings.

Reply:
Understood, thanks!